routes/documents.js

```diff
@@ -2,56 +2,61 @@ const express = require('express');
 
 const router = express.Router();
 const Document = require('../models/document');
+const User = require('../models/user');
 const { DocumentBodySchema } = require('./validators/validators');
 const validator = require('express-joi-validator');
 
-router.get('/', (req, res, next) => {
-  Document.find({})
-    .select('-content')
-    .then((document) => {
-      res.send(document);
+const sendError = res =>
+  res.status(401).send({
+    message: 'User is not logged!',
+    status: 401,
+  });
+
+
+const isAuthenticated = (req, res, next) =>
```

Comment:
This is correct usage. Please use it everywhere where authentication is required. You can extract this function outside of this class.

Also, to prevent searching for user again in business logic, you can assign here found user to request object like this:
`req.user = user`